src/dev/run_prettier.js

```diff
@@ -0,0 +1,25 @@
+import { resolve, dirname } from 'path';
+
+import getopts from 'getopts';
+
+const ROOT = dirname(require.resolve('../../package.json'));
+const flags = getopts(process.argv.slice(2));
+
+if (!flags.config) {
+  process.argv.push('--config', resolve(ROOT, '.prettierrc'));
+}
+
+if (!flags.ignorePath) {
+  process.argv.push('--ignore-path', resolve(ROOT, '.prettierignore'));
+}
+
+if (!flags._.length) {
+  process.argv.push('./**/*.js');
+}
+
+if (!flags.write) {
```

Comment:
Does this allow `--no-write` to work here? Might have to check for the `flags.noWrite`, or `=== false`

Reply:
@spalger I don't see the need to support that. This command is just to let a team convert their entire project in one PR vs how editor plugins work converting 1 file at a time.